index.html

```diff
@@ -8,279 +8,192 @@
+          identifiers</a> using <a data-cite=
+          "!URL#concept-url-equals">equals</a>. [[!URL]]
         </p>
-        <ul>
-          <li>Let <em>A</em> be the first payment method identifier string and
-          let <em>B</em> be the second payment method identifier string.
-          </li>
-          <li>Let <em>urlA</em> be the result from the <a>URL parser</a> when
-          parsing <em>A</em> with no <a>base URL</a>.
-          </li>
-          <li>If either <em>urlA.query</em> or <em>urlA.fragment</em> are not
-          <em>null</em> terminate the algorithm with a result of <em>no
-          match</em> and discard <em>A</em> from further matching.
-          </li>
-          <li>Let <em>urlB</em> be the result from the <a>URL parser</a> when
-          parsing <em>B</em> with no <a>base URL</a>.
-          </li>
-          <li>If either <em>urlB.query</em> or <em>urlB.fragment</em> are not
-          <em>null</em> terminate the algorithm with a result of <em>no
-          match</em> and discard <em>B</em> from further matching.
-          </li>
-          <li>The identifiers match if <em>urlA</em> <strong>equals</strong>
-          <em>urlB</em> using the <a>URL equivalence</a> test (i.e. the test
-          returns true).
-          </li>
-        </ul>
       </section>
     </section>
     <section>
       <h2>
-        W3C-Defined Strings as Payment Method Identifiers
+        Standardized payment method identifiers
       </h2>
       <p>
-        W3C expects to publish a small number of payment method specifications
-        that define abstractions across similar payment methods (such as credit
-        card payments, tokenized payments, or credit transfers). W3C expects
-        these payment methods will be implemented in a wide variety of third
-        party payment apps. W3C identifies these payment methods with (short)
-        strings; W3C expects to mint these strings in payment method
-        specifications published by the Web Payments Working Group.
+        A <dfn data-lt="standardized payment method identifiers">standardized
+        payment method identifier</dfn> is a string that represents a
+        <a>standardized payment method</a>.
+      </p>
+      <p>
+        User agents MAY support zero or more <a>standardized payment method
+        identifiers</a> listed in section <a href="#registry"></a>.
       </p>
       <section>
         <h2>
-          Syntax
+          Validity
         </h2>
+        <p class="note">
+          When minting a new <a>standardized payment method identifier</a> for
+          the purpose of standardization, be sure that it conforms to the
+          following regular expression: <code>[a-z0-9-]+</code>.
+        </p>
         <p>
-          The syntax of W3C strings used as payment method identifiers is
-          constrained by this regular expression:
+          User agents MUST NOT enforce validity or well-formedness of
```

Comment:
This seems a bit confusing to me at first read. All we're trying to say is "methods either match (great) or they don't (ignored)." It took me a bit to understand what this meant. Example might help?